library/Vanilla/Utility/ContainerUtils.php

```diff
@@ -67,9 +67,33 @@ public static function currentTheme(): ReferenceInterface {
     public static function cacheBuster(): ReferenceInterface {
         return new Callback(
             function (ContainerInterface $dic) {
-                $cacheBuster = $dic->get(\Vanilla\Web\Asset\DeploymentCacheBuster::class);
+                $cacheBuster = $dic->get(DeploymentCacheBuster::class);
                 return $cacheBuster->value();
             }
         );
     }
+
+    /**
+     * Replace container instance of a class, if present, and optionally add it as an alias for its replacement.
+     *
+     * @param Container $container
```

Comment:
I'm trying to get us in the habit of doing a bit more documentation in utility function doc blocks. This is a good candidate because it's not immediately obvious to me what this method does. In particular the `$target` and `$replacement` parameters should get a good sentence to say that they are rules (class name, alias, or custom rule).

Reply:
An example in the doc block that explains the problem it solves too.